.github/workflows/test.yml

```diff
@@ -62,6 +62,32 @@ jobs:
         env:
           PERCY_TOKEN: ${{ secrets.PERCY_TOKEN }}
 
+  chromatic-deployment:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x]
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # 👈 Required to retrieve git history
+      - name: Install dependencies
+        run: yarn install --immutable
+      - name: Build Storybook 📚
+        run: yarn workspace @chanzuckerberg/eds-components run build:storybook
+      - name: Run Accessibility Tests 💛
+        run: yarn workspace @chanzuckerberg/eds-components run storybook:axeOnly
```

Comment:
Is this copy pasta and can be removed?

Reply:
Was intentionally added so that if we were to remove Percy, we can just remove the code block and would make more sense in terms of commit history, but makes sense to remove since it is double testing. What is your advice?